### DISCUSSION TITLE
Is there an easy fix to deleting multiple projects?

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#575] [Q&A] [answered]
I have a ton of project files, but I would like to remove them. However, removing them in the front-end is a bit tedious for many files. Is there a workaround?

**Comment by terrymyc** [comment]
Unfortunately, there is no explicit way to delete multiple projects once in the front end. You may issue a feature request and we will discuss the implementation of the feature.

**Comment by J535D165** [comment, marked as answer]
There is a hidden `.asreview` folder in your home directory. Each folder is a project. You can delete the folders and the projects are removed. 

DISCLAIMER: This isn't officially supported and should be handled with care.

**Reply by SagevdBrand** [reply]
Many thanks!
For windows users, you can probably find it somewhere here: `C:\Users\X\.asreview`, with X being your name for example.
